conf/logback.xml

```diff
@@ -49,6 +49,8 @@
         <appender-ref ref="STDOUT"/>
     </logger>
 
+    <logger name="api-subscription-fields" level="${logger.api-subscription-fields:-INFO}"/>
+
     <root level="INFO">
```

Comment:
replace to 
```
<root level="${logger.application:-INFO}">
        <appender-ref ref="STDOUT"/>
    </root>
```
See here: https://github.com/hmrc/api-gateway/blob/master/conf/application-json-logger.xml#L13-L15